### DISCUSSION TITLE
Contributors Trello

### DISCUSSION THREAD

**Discussion by Poseidon281** [pelican-dev/panel#58] [Ideas]
I thought maybe it is a good idea to make a Trello Page or Github Project. To make it public in viewer mode for everyone so that people can see what still needs to be done and for (trusted) contributors to make them editors in Trello so they can help with the project

**Comment by ItsNeil17** [comment]
+1 GitHub issue tracking isn't the best but it works. 

Setting up even a GitHub project would be better than just the Issues tab.

**Comment by lancepioch** [comment]
Thanks, we're working on using Github milestones!
